pkg/pipelines/bootstrap.go

```diff
@@ -405,29 +398,29 @@ func checkPipelinesFileExists(appFs afero.Fs, outputPath string, overWrite bool)
 func createInitialFiles(fs afero.Fs, repo scm.Repository, o *BootstrapOptions) (res.Resources, error) {
 	cicd := &config.PipelinesConfig{Name: o.Prefix + "cicd"}
 	pipelineConfig := &config.Config{Pipelines: cicd}
-	pipelines := createManifest(repo.URL(), pipelineConfig)
+	manifest := createManifest(repo.URL(), pipelineConfig)
 	initialFiles := res.Resources{
-		pipelinesFile: pipelines,
+		pipelinesFile: manifest,
```

Comment:
We changes between pipelines <-> manifest many times.  (Sigh)

Reply:
Yeah, the result of calling `createManifest` is stored in `manifest` and we have `manifestFile`.

I suspect we could take another pass at naming here.